src/draggable.js

```diff
@@ -9,18 +9,16 @@ export default class Draggable {
     }
 
     onMouseDown(e) {
-        if (!this.start) {
-            this.start = e;
-        }
-
+        this.start = e;
+        this.item.setPreviousPosition()
         document.addEventListener('mousemove', this.onMouseMove);
         document.addEventListener('mouseup', this.onMouseUp);
     }
 
-    onMouseMove({ clientX, clientY }) {
+    onMouseMove(e) {
         this.item.setPosition(
-            clientX - this.start.clientX,
-            clientY - this.start.clientY,
+            this.item.previousX ? this.item.previousX + e.clientX - this.start.clientX : e.clientX - this.start.clientX,
+            this.item.previousY ? this.item.previousY + e.clientY - this.start.clientY : e.clientY - this.start.clientY
```

Comment:
I guess u can also simplify this to split calculations and ternary operator

Reply:
fixed